agimus_controller/tests/test_ocp_croco_base.py

```diff
@@ -108,26 +108,28 @@ def set_reference_weighted_trajectory(self, reference_weighted_trajectory):
     def setUpClass(self):
         # Mock the RobotModelFactory and OCPParamsCrocoBase
         robot = robex.load("panda")
-        urdf_path = Path(robot.urdf)
-        srdf_path = Path(robot.urdf.replace("urdf", "srdf"))
-        urdf_meshes_dir = urdf_path.parent.parent.parent.parent.parent
+        urdf = Path(robot.urdf)
+        srdf = Path(robot.urdf.replace("urdf", "srdf"))
+        urdf_meshes_dir = urdf.parent.parent.parent.parent.parent
         free_flyer = False
         locked_joint_names = ["panda_finger_joint1", "panda_finger_joint2"]
         reduced_nq = robot.model.nq - len(locked_joint_names)
-        full_q0 = np.zeros(robot.model.nq)
-        q0 = np.zeros(reduced_nq)
+        moving_joint_names = set(robot.model.names) - set(
+            locked_joint_names + ["universe"]
+        )
+        q0 = np.zeros(robot.model.nq)
         armature = np.full(reduced_nq, 0.1)
+
         # Store shared initial parameters
         self.params = RobotModelParameters(
             q0=q0,
-            full_q0=full_q0,
             free_flyer=free_flyer,
-            locked_joint_names=locked_joint_names,
-            urdf=urdf_path,
-            srdf=srdf_path,
+            moving_joint_names=moving_joint_names,
+            urdf=urdf,
+            srdf=srdf,
```

Comment:
We said we would keep the variables that you create with the proper suffix to make the code reader understand that we send the path here.
```suggestion
            urdf=urdf_path,
            srdf=srdf_path,
```